sympy/polys/rootoftools.py

```diff
@@ -221,17 +221,17 @@ class ComplexRootOf(RootOf):
 
     After evaluation it is now
 
-    >>. r._get_interval()
+    >>> r._get_interval() # doctest: +SKIP
     (-165/169, -206/211)
```

Comment:
These methods should be internal and not documented I think (not necessarily to be solved in this PR)

Reply:
It's OK to document internal methods. The documentation can be helpful for developers working on the code. The existence of a docstring or even a doctest shouldn't imply that something is public. 

But we probably should be more particular about what goes in Sphinx. I added a lot of internal things to Sphinx because other things were mentioning them in See Also. I agree we should determine that in a separate issue.

Reply:
These particular methods are problematic because they can be affected by cache etc. I presume that's why you have `+SKIP` here.

Reply:
Yes. Actually all I did here was fix the formatting. With `>>.` Sphinx doesn't format it correctly. I made it so the doctest is still not run (I'm assuming the `>>.` was on purpose), but it actually renders correctly.